pkg/eosclientgrpc/eos_grpc/eos_grpc.pb.go

```diff
@@ -0,0 +1,3182 @@
+// Copyright 2018-2020 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: eos_grpc.proto
```

Comment:
could you add a link to the source?

Reply:
is it this one: https://gitlab.cern.ch/eos/grpc-proto/-/blob/master/protobuf/Rpc.proto ?

Reply:
Hi, the source is in the same directory by now, with a comment showing the command line used to compile it.